server/app.js

```diff
@@ -27,4 +28,9 @@ app.use((_req, res) => {
   res.sendFile(join(__dirname, '..', 'client', 'build', 'index.html'));
 });
 
+// eslint-disable-next-line no-unused-vars
+app.use((err, req, res, _next) => {
+  res.send({ statusCode: err.statusCode, error: errorHandle(err.statusCode) });
```

Comment:
What if the error didn't have a `statusCode`? then the status code that'll be sent won't be `500`
I think it's better if you do the work for handling the error here, and no need for `errorHandle` middleware